packages/medusa/src/services/idempotency-key.ts

```diff
@@ -4,7 +4,10 @@ import { TransactionBaseService } from "../interfaces"
 import { DeepPartial, EntityManager } from "typeorm"
```

Comment:
Suggestion: @srindom the workstage when trying to update the ikey, it will try to retrieve it. Should we directly update without retrieving as at that moment it should be sure that we are holding an existing ikey and would remove one more step. Would like your opinion